vectors.txt

```diff
@@ -19756,12599 +17902,3 @@ msg_1_ciphertext=131094feaa8cbacb6c348050711162cf8d44b13e8de882c98410e7d3981d51
-msg_0_payload=
-msg_0_ciphertext=358072d6365880d1aeea329adf9121383851ed21a28e3b75e965d0d2cd166254098cdd7308257dd69233ab7d725ee8bc
-msg_1_payload=
-msg_1_ciphertext=64b101b1d0be5a8704bd078f9895001fc03e8e9f9522f188dd128d9846d484668edb3682137234dad14a8b4c8866e03d
-msg_2_payload=79656c6c6f777375626d6172696e65
-msg_2_ciphertext=7cb6cbbbddfefdc4d030ab94ab2aab13422ef4d87e054a08bb10da446c6f36
-msg_3_payload=7375626d6172696e6579656c6c6f77
-msg_3_ciphertext=e3926ab002f3c5051150d49e11388040a8397644b461aed00fad63e8023241
-
-handshake=Noise_NKpsk1_25519_ChaChaPoly_SHA512
-resp_static=0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20
-gen_init_ephemeral=202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f
-gen_resp_ephemeral=4142434445464748494a4b4c4d4e4f505152535455565758595a5b5c5d5e5f60
-preshared_key=2176657279736563726574766572797365637265747665727973656372657421
-msg_0_payload=
-msg_0_ciphertext=358072d6365880d1aeea329adf9121383851ed21a28e3b75e965d0d2cd166254cbd6c6755faedda9da18ea3a14dbb128
-msg_1_payload=
-msg_1_ciphertext=64b101b1d0be5a8704bd078f9895001fc03e8e9f9522f188dd128d9846d484664c48eb806966bdd24e98fe421b3d4815
-msg_2_payload=79656c6c6f777375626d6172696e65
-msg_2_ciphertext=1193e531aed0c1f9c475668bc5420beb3ee15f3520931d4a283185763289fa
-msg_3_payload=7375626d6172696e6579656c6c6f77
-msg_3_ciphertext=d4b9ff958655454654452cfc613b0b9a7fd7dac8aae5f1f0382edba43b0d1a
-
-handshake=Noise_NKpsk2_25519_ChaChaPoly_SHA512
-resp_static=0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20
-gen_init_ephemeral=202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f
-gen_resp_ephemeral=4142434445464748494a4b4c4d4e4f505152535455565758595a5b5c5d5e5f60
-preshared_key=2176657279736563726574766572797365637265747665727973656372657421
-msg_0_payload=
-msg_0_ciphertext=358072d6365880d1aeea329adf9121383851ed21a28e3b75e965d0d2cd16625486bf3bfeb6861b365528620b68992d7b
-msg_1_payload=
-msg_1_ciphertext=64b101b1d0be5a8704bd078f9895001fc03e8e9f9522f188dd128d9846d484668acd99b4cbc7ce0c14ff2be16afc02bf
-msg_2_payload=79656c6c6f777375626d6172696e65
-msg_2_ciphertext=65fd72393fd5065a54fa40313eaf8b7a052631aaf3f7368622f909aff8f4e8
-msg_3_payload=7375626d6172696e6579656c6c6f77
-msg_3_ciphertext=28de903b3a3f50fd29d23549b726ddeb4c16f7db17c1757b4dd8d54ecfd8a2
-
-handshake=Noise_NK_25519_ChaChaPoly_SHA512
-resp_static=0102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f20
-gen_init_ephemeral=202122232425262728292a2b2c2d2e2f303132333435363738393a3b3c3d3e3f
-gen_resp_ephemeral=4142434445464748494a4b4c4d4e4f505152535455565758595a5b5c5d5e5f60
-msg_0_payload=746573745f6d73675f30
-msg_0_ciphertext=358072d6365880d1aeea329adf9121383851ed21a28e3b75e965d0d2cd16625490f8f004794122bca779d9bedcc39b7bfb88883f6a56029320aa
-msg_1_payload=746573745f6d73675f31
-msg_1_ciphertext=64b101b1d0be5a8704bd078f9895001fc03e8e9f9522f188dd128d9846d48466e93b0314d9d7741d4e27466c0dd7656cf06185f352ffe422a67d
-msg_2_payload=79656c6c6f777375626d61726
```

Comment:
Any idea what happened to all of these test vectors?

Reply:
I removed all the text vectors associated to the `XR` pattern :o did I mess up with something?

Reply:
This one looks like it was a NK pattern?

Reply:
gosh yeah, looks like my regex was flawed. Let me try again